queue_and_stack/dll_queue.py

```diff
@@ -1,19 +1,30 @@
-import sys
-sys.path.append('../doubly_linked_list')
+# import sys
+# sys.path.append('../doubly_linked_list')
 from doubly_linked_list import DoublyLinkedList
 
 
 class Queue:
     def __init__(self):
         self.size = 0
         # Why is our DLL a good choice to store our elements?
-        # self.storage = ?
+
+        # -> Because LLs are O(1) inserting and deleting nodes from head and tail. And that is what a queue is about:
+        # adding values to head and removing from tail
+
+        self.storage = DoublyLinkedList()
 
     def enqueue(self, value):
-        pass
+        self.size += 1
+        self.storage.add_to_head(value)
```

Comment:
To better drive home the concept of a Queue it's best to add to `tail` that is add an item to the last node of the `linked list` and remove from `head` that is the first node.